rust/agama-software/src/callbacks/commit_download.rs

```diff
@@ -11,6 +11,8 @@ use crate::callbacks::ask_software_question;
 
 #[derive(Clone)]
 pub struct CommitDownload {
+    // FIXME: field is never read
```

Comment:
yeap, we plan to use it when we decide how exactly we want to report download and install progress.